Filters/exceptions.txt

```diff
@@ -1,3 +1,6 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/150038
+! Blocked by ||getblueshift.com^
+@@||clickattr.wayup.com^
```

Comment:
Maybe it would be a good idea to add `|` at the end (like it is in other rules).
```adblock
@@||clickattr.wayup.com^|
```